src/components/Contacts/ContactDetails/ContactTasksTab/ContactTaskRow/TaskCompleteButton/TaskCompleteButton.tsx

```diff
@@ -22,6 +22,9 @@ const ButtonWrap = styled(Button, {
     backgroundColor: theme.palette.mpdxGreen.main,
     color: theme.palette.common.white,
   },
+  ['@media (max-width:500px)']: {
```

Comment:
You don't need the brackets. Brackets are for when they key is an expression instead of a string, like `const obj = { ['key' + index]: 'value' };`

```suggestion
  '@media (max-width:500px)': {
```